taxonomies/ingredients.txt

```diff
@@ -17958,19 +17958,16 @@ wikipedia:en:https://en.wikipedia.org/wiki/Olive_pomace_oil
 
 <en:vegetable oil and fat
 en:rapeseed oil, rapeseed fat, rapeseed oil and fat
-ca:Oli de colza
 cs:řepkový olej, olej řepkový
 da:rapsolie
 de:Rapsöl, Rapsol
 es:aceite de nabina, aceite vegetal de nabina
 et:rapsiõli, rapsiseemneõli
 fi:rapsiöljy, rapsirasva, rapsiöljy ja -rasva
-fr:Huile de colza
```

Comment:
we should add "fr:Huile de colza ou de navette" as the French name of en:rapeseed oil